### PR TITLE
Export react-search to unstable

### PR DESCRIPTION
**Pull request by emmayjiang**
This PR exports react-search to unstable, as part of #26648.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount_base.html">73</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount.html">76</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Avatar-mount_base.html">574</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Avatar-mount.html">593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Button-mount_base.html">291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Button-mount.html">287</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Field-mount_base.html">1090</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Field-mount.html">1075</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProvider-mount_base.html">645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProvider-mount.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount_base.html">73</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount.html">76</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">64</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">67</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">72</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">71</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/InfoButton-mount_base.html">8</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/InfoButton-mount.html">8</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/MakeStyles-mount_base.html">870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/MakeStyles-mount.html">858</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Persona-mount_base.html">1669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/Persona-mount.html">1627</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/SpinButton-mount_base.html">1315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28464/perf-test-react-components/SpinButton-mount.html">1319</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4bc71e1e82a2b0044ffdde84f84d8c47271bbdac'>4bc71e1e82a2b0044ffdde84f84d8c47271bbdac</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-search/stories/SearchBox/SearchBoxBestPractices.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 ### Do
 
+- **Consider using `SearchBox` with underline or outline appearances.** When the contrast ratio against the immediate surrounding color is less than 3:1, consider using underline or outline styles which has a bottom border stroke. But please ensure the color of bottom border stroke has a sufficient contrast which is greater than 3 to 1 against the immediate surrounding.
```
Comment (smhigley) [inline packages/react-components/react-search/stories/SearchBox/SearchBoxBestPractices.md:5]: This isn't something for this PR, but it'd be a nice quick change to add an item here for something like this:
> - **Consider wrapping SearchBox in a `role="search"` container for top-level site or app search fields.** This role exposes the search field as a landmark, and makes it easier to find for screen reader users. It should not be used unless the SearchBox is the primary search area for the entire site or app.

File: packages/web-components/docs/api-report.md [hunk 2]
```diff
@@ -228,7 +228,7 @@ export const allComponents: {
     fluentTooltip: (overrideDefinition?: OverrideFoundationElementDefinition<FoundationElementDefinition> | undefined) => FoundationElementRegistry<FoundationElementDefinition, Tooltip>;
     fluentTreeView: (overrideDefinition?: OverrideFoundationElementDefinition<FoundationElementDefinition> | undefined) => FoundationElementRegistry<FoundationElementDefinition, typeof TreeView>;
     fluentTreeItem: (overrideDefinition?: OverrideFoundationElementDefinition<TreeItemOptions> | undefined) => FoundationElementRegistry<TreeItemOptions, Constructable<FoundationElement>>;
-    register(container?: Container | undefined, ...rest: any[]): void;
+    register(container?: Container, ...rest: any[]): void;
```
Comment (smhigley) [inline packages/web-components/docs/api-report.md:231]: Is this change intentional?

File: change/@fluentui-react-components-bc19f521-d33b-4fa7-bfa8-a30b797d6b83.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "to unstable",
+  "packageName": "@fluentui/react-components",
```
Comment (smhigley) [inline change/@fluentui-react-components-bc19f521-d33b-4fa7-bfa8-a30b797d6b83.json:4]: Looks like there are double change files somehow for both react-components and react-search

**Review by smhigley — COMMENTED** [review]
Looks like there's something up with the changefiles, and a random other change that snuck in (maybe with a top-level build?), but 🎉 for SearchBox!
